docs/ZendeskMultiObjects-batchsource.md

```diff
@@ -24,14 +24,15 @@ Configuration
 **Admin Email:** Zendesk admin email.
```

Comment:
The description for this plugin doesn't match the Google doc. The Zendesk Batch Source markdown has the updated overview. https://docs.google.com/document/d/1T8eCPI9rgZm7tL9OfhWb2EtUGfdhMYY7aryfos7deKM/edit?resourcekey=0-JCNeuQYONXE8cB9WS3iEJw#

Also, the name of the plugin should be Zendesk Multi Objects Batch Source. The current name reads without a space Zendesk MultiObjects Batch Source

Reply:
done